analysis/study_utils.py

```diff
@@ -213,3 +213,25 @@ def redact_events_table(events_counts, low_count_threshold, rounding_base):
     )
 
     return events_counts
+
+
+def convert_weekly_to_monthly(counts_table):
```

Comment:
Could you add a docstring to say that we're grouping into periods of four weeks, instead of one month?